src/AtmosphereModels/radiation_interface.jl

```diff
@@ -0,0 +1,17 @@
+#####
+##### Radiation interface for AtmosphereModel
+#####
+##### This file defines stub functions that are implemented by radiation extensions
+##### (e.g., BreezeRRTMGPExt).
+#####
+
+"""
+    update_radiation!(radiation, model)
```

Comment:
We don't like `$(TYPEDSIGNATURES)` anymore?  Most of the docstrings added over the last few days are manually written like this one